pkgs/development/nim-packages/ws/default.nix

```diff
@@ -0,0 +1,17 @@
+{ lib, buildNimPackage, fetchFromGitHub }:
+
+buildNimPackage (final: {
+  pname = "ws";
+  version = "0.5.0";
+  src = fetchFromGitHub {
+    owner = "treeform";
+    repo = "ws";
+    rev = final.version;
+    hash = "sha256-3wVi6CjMTjcc5MJEozJN6W3TuYDb53w2MDCsv6lMH0k=";
+  };
+  preCheck = "rm tests/test_ws.nim tests/test_timeout.nim";
+  meta = final.src.meta // {
```

Comment:
Please copy the meta that src provides. It makes it a lot easier for other people to discover the homepage.

Also those packages do not seem to have meta.maintainer which is not allowed according to the contributing guide.